test/Test/Database/LSMTree/Internal/RawPage.hs

```diff
@@ -293,8 +297,8 @@ prop_single_entry k op blobref = label (show $ BS.null sfx) $
     sfxSize :: Int
     sfxSize = BS.length sfx
 
-bsToVector :: BS.ByteString -> P.Vector Word8
-bsToVector = P.fromList . BS.unpack
+bsToRawBytes :: BS.ByteString -> RawBytes
```

Comment:
There is function for this and `bsFromRawBytes`, in `RawBytes` module, isn't there?

If there aren't, there probably should be.